projects/packages/stats-admin/src/class-dashboard.php

```diff
@@ -284,4 +290,38 @@ protected static function get_site_locale() {
 		return $locale;
 	}
 
+	/**
+	 * Get the features of the current plan.
+	 */
+	protected function get_plan_features() {
+		if ( ! class_exists( 'Jetpack_Plan' ) ) {
+			return array();
+		}
+		$plan = \Jetpack_Plan::get();
+		if ( empty( $plan['features'] ) ) {
+			return array();
+		}
+		return $plan['features'];
+	}
+
+	/**
+	 * Get the capabilities of the current user.
+	 *
+	 * @return array An array of capabilities.
+	 */
+	protected function get_current_user_capatibilites() {
+		// Feature lock.
+		if ( ! isset( $_GET['flags'] ) || $_GET['flags'] !== 'stats/ads-page' ) {// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			return array();
+		}
+		if ( version_compare( Main::VERSION, '0.4.0-alpha', '<' ) ) {
```

Comment:
Why do we need this check here?

Reply:
Hi @fgiannar Good catch! This is useless... I was hoping to only show the Ads page for version >= `0.4.0-alpha` but which would always is. The check needs to be done in the front end, or a front end version bump.

Reply:
Removed in 92fe0a0e14